index.bs

```diff
@@ -253,18 +253,251 @@ Control Protocols {#control}
 Issue(77): Describe CBOR based mechanism for
 exchanging control messages, results, and events.
 
-Presentation API Protocol {#presentation-api}
+
+Presentation Protocol {#presentation-protocol}
 ---------------------------------------------
 
-Issue(55): Incorporate material from the
-[Protocol](protocol.md) document.
+This section defines the use of the Open Screen Protocol for starting,
+stopping, and controlling presenetations as defined by
+[[PRESENTATION-API|Presentation API]].  A subsequent section will
+define how APIs in [[PRESENTATION-API|Presentation API]] map to the
+protocol messages defined in this section.
+
+For all messages defined in this section, see Appendix A for the full
+CDDL definitions.
+
+<!-- TODO: Add a capability that indicates support for the
```

Comment:
Can you file this as a GitHub issue, to define a capabilities structure? Or I can do a sweep later to file TODOs in the spec.

Reply:
OK

https://github.com/webscreens/openscreenprotocol/issues/123